pandas/core/arrays/string_.py

```diff
@@ -290,7 +290,7 @@ def _reduce(self, name, skipna=True, **kwargs):
     def value_counts(self, dropna=False):
         from pandas import value_counts
 
-        return value_counts(self._ndarray, dropna=dropna).astype("Int64")
+        return value_counts(self._ndarray, dropna=dropna)
```

Comment:
Can you revert this back to what it was on master (will need to change the string tests back as well)?

Reply:
If revert, value_counts method return different dtype and need to change many tests. ([comment](https://github.com/pandas-dev/pandas/pull/33674#issuecomment-619389810))

Reply:
@dsaxton why/where were these changed to Int64 in the first place?

Reply:
It looks like it was added here https://github.com/pandas-dev/pandas/pull/30824

Reply:
@jreback this is converted to Int64 on purpose, in the idea that operations on nullable columns should, to the extent possible, also result in new nullable dtypes (similarly as doing a comparison operation with string dtype will give a nullable boolean dtype)